api/src/main/java/org/openmrs/module/appointmentscheduling/api/impl/AppointmentServiceImpl.java

```diff
@@ -371,64 +371,64 @@ public List<AppointmentBlock> getOverlappingAppointmentBlocks(
 
 	@Override
 	@Transactional(readOnly = true)
-	public List<Appointment> getAllAppointments() {
+	public List<AppointmentData> getAllAppointmentDatas() {
```

Comment:
Probably want to keep all these as getAllAppointments() etc, instead of adding the "data"

Reply:
That's _probably_ ok, but I'm not 100% sure.  I started out that way, trying to keep things as backwards-compatible as possible, but then got hit with an issue with the RequiredDataAdvice, which matches on method name and object class to determine if a save / void / etc. is happening and adding side effects.  So I was a bit worried about inadvertently removing expected behavior.  It certainly might be better to try to identify this behavior and then ensure the service methods respect it, and leave the method names in the service along though.  We can discuss @mogoodrich

Reply:
Ugh, okay... maybe you are convincing me a little more that just having two distros and not allowing them to run in parallel might be the way to go.